extensions/amp-story/1.0/amp-story.js

```diff
@@ -592,10 +592,6 @@ export class AmpStory extends AMP.BaseElement {
    * @private
    */
   rewriteStyles_(styleEl) {
-    if (!isExperimentOn(this.win, 'amp-story-responsive-units')) {
-      return;
-    }
-
```

Comment:
Related to #31595. This file was not handled by the automated sweep tool since it contains what it considers a syntax error:

https://github.com/ampproject/amphtml/blob/3514120627396355c4c5974f30ee023f6f51a21f/extensions/amp-story/1.0/amp-story.js#L117

`TODO` for myself: Ensure that the sweep tool uses a babel configuration that's somewhat uniform with the runtime ("somewhat" since it can't resolve all dynamic transforms.)

Reply:
I think the missing ingredients are the `jsx` and `importAssertions` plugins for babel (see 
https://github.com/ampproject/amphtml/issues/31756#issuecomment-754928751). Is there a way to make the parser in `build-system/tasks/sweep-experiments/jscodeshift/remove-experiment-config.js` use these two plugins?

Reply:
Nice, thanks! Fixed on #31864